OTRExporter/OTRExporter/RoomExporter.cpp

```diff
@@ -207,12 +207,12 @@ void OTRExporter_Room::Save(ZResource* res, const fs::path& outPath, BinaryWrite
 				Declaration* dListDeclXlu = poly->parent->GetDeclaration(GETSEGOFFSET(test->xlu));
 
 				if (test->opa != 0)
-					writer->Write(StringHelper::Sprintf("%s\\%s", OTRExporter_DisplayList::GetParentFolderName(res).c_str(), dListDeclOpa->varName.c_str()));
+					writer->Write(StringHelper::Sprintf("%s/%s", OTRExporter_DisplayList::GetParentFolderName(res).c_str(), dListDeclOpa->varName.c_str()));
```

Comment:
I'm wondering if there is a better cross platform way to handle this? I suppose a "/" should work on win32 and linux. But IIRC there are certain places where win32 needs "\". Is there something like Python's `os.path.sep` in this C++ env? Or does this not matter at all, and "/" should just work fine? (I'm hoping it's the later case).

Reply:
I actually don't know the answer, this was just a temporary fix until I test the PR on windows and confirm that paths specified in mpq files work on both linux and windows.
Once I can test that, I will actually make a more robust fix.